internal/test/wmcb/main_test.go

```diff
@@ -1,19 +1,28 @@
 package wmcb
 
 import (
-	e2ef "github.com/openshift/windows-machine-config-operator/internal/test/framework"
+	"log"
 	"os"
 	"testing"
+
+	e2ef "github.com/openshift/windows-machine-config-operator/internal/test/framework"
 )
 
 // framework holds the instantiation of test suite being executed. As of now, temp dir is hardcoded.
-// TODO: Create a temporary remote directory on the Windows node
-var framework = &e2ef.TestFramework{RemoteDir: "C:\\Temp"}
+var (
+	framework = &e2ef.TestFramework{}
+	// vmCount is the number of VMs the test suite requires
```

Comment:
How do you want to expose this to end-user? Some command line flag?

Reply:
I will add a TODO for this. I don't want to increase the complexity of the PR with that change too.